src/pages/cli/graphql/authorization-rules.mdx

```diff
@@ -51,7 +51,7 @@ Use the guide below to select the correct authorization strategy for your use ca
 | **Recommended use case** | **Strategy** | **Provider** |
 |---|---|---|
 | Public data access. Anyone with the AppSync API Key should have access. | [`public`]() | `apiKey` |
```

Comment:
Change this line to: "Public data access where users or devices are anonymous. Anyone with the AppSync API key is granted access"